submissions/QuickPerformance/QuickPerformance.md

```diff
@@ -0,0 +1,84 @@
+
+### Network Speed Issues
+
+Sometimes, your network may not be the best. That can be due to many things itself, such as maintenance from your ISP or a power outage of your DNS's datacenters. There are a few utilities that can however help mitigate these sorts of issues.
+
+To explain what Factorio requires, that is a long*er* process. However, *cube* has explained it largely in [FFF#136](https://factorio.com/blog/post/fff-136). Wube have done non-standard things to let the game work, such as re-implementing large parts of TCP in UDP (TCP and UDP are types of communication of packets. TCP makes sure that everything is confirmed and arrives properly, UDP basically "doesn't care". Common UDP uses are in torrenting, as you can get other parts of files later on. TCP is used for many day-to-day things, such as web browsing) so the player wouldn't need to have two ports opened and NAT punching would work (NAT punching is a thing that is used for your router/modem to send Factorio traffic properly, explained well in [FFF#143](https://factorio.com/blog/post/fff-143))
+
+Firstly, you can try a quick speed test, to check if your network is fast enough, from sites such as [Ookla's Speedtest](https://speedtest.net) or [fast.com (Netflix)](https://fast.com). Both sites will measure similar data, so pick your poison. There are however 3 things that matter (referencing Ookla's Speedtest):
+- Download speed
+- Upload speed
+- Latency
+
+![Speedtest.net result for reference](SpeedtestResult.png)
+{% include image.html src="SpeedtestResult.png" alt="Speedtest.net result for reference" }
+
+For Factorio, your download and upload speeds don't need to be very high, as the map (savefile) is sent over to you when you connect to a server. The speed of the map downloading depends on your download speed and the server's upload speed; as you can't download something faster than it is being uploaded. Generally, savefiles are within 50MB (if you don't run with seriously modded games), so on an [average connection](https://www.speedtest.net/global-index), a save would be downloaded within 15 seconds. It doesn't however matter how fast a save downloads, since it is just a file transfer in the end. The speed of your download however affects for how many [ticks](https://wiki.factorio.com/Time#Ticks) (in-game unit of time, 60 per second with 100% performance) you will however be catching up (the part of loading the map after you download it). Generally speaking, anything *should* be fine, but mileage may vary. Another thing from Ookla's speedtest is your ping (latency) to their datacenter. That is not always accurate in respect towards a Factorio server, as Factorio servers can be anywhere in the world, whilst Ookla connects you to your *closest* datacenter.
```

Comment:
This paragraph could do with being split up just before you start talking about ping stuff. I'd also consider adding a little bit more information about what exactly catching up is and some scenarios where you may not be able to catch up (big maps on very slow connections *speaking from experience :p*) and how you can help mitigate that by pausing whilst someone joins.